packages/react-kit/src/hooks/erc/erc721.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from "react-query";
+import { useCoreSDKWithContext } from "../useCoreSdkWithContext";
+
+export const useErc721OwnerOf = (
+  {
+    contractAddress,
+    tokenIds
+  }: {
+    contractAddress: string | undefined;
+    tokenIds: (string | null | undefined)[] | undefined;
+  },
+  { enabled }: { enabled: boolean | undefined }
+) => {
+  const coreSDK = useCoreSDKWithContext();
+  return useQuery(["erc721-owner-of", coreSDK.uuid], () => {
```

Comment:
coreSDK.uuid does not exist in the main branch (commit widget branch not merged yet)

Reply:
Sorry, my mistake